server/src/testflinger/templates/job_detail.html

```diff
@@ -25,59 +25,133 @@ <h1 class="p-heading--3">Job Detail - {{ job.job_id }}</h1>
       </tr>
     </tbody>
   </table>
-  <h2 class="p-muted-heading">Job Definition</h2>
-  <div class="p-code-snippet">
-    <div class="p-code-snippet__header">
-      <h5 class="p-code-snippet__title">Provision Data</h5>
+  <h2 class="p-muted-heading" id="job-definition">
+    Job Definition
+    <a href="#job-definition"
+       class="p-icon--anchor"
+       aria-label="Link to Job Definition section">#</a>
+  </h2>
```

Comment:
The class should not be `p-icon--anchor`, but `p-link--anchor-heading`. Additionally, the `<a>` element should contain the text. This will give us the Vanilla anchor heading style (normal text, but adds a link icon on hover)


```suggestion
  <h2 class="p-muted-heading" id="job-definition">
    <a href="#job-definition"
       class="p-link--anchor-heading"
       aria-label="Link to Job Definition section">Job Definition</a>
  </h2>
```